modules/mbedtls/SCsub

```diff
@@ -42,13 +43,14 @@ if env["builtin_mbedtls"]:
         "entropy_poll.c",
         "error.c",
         "gcm.c",
-        "havege.c",
+        "godot_core_mbedtls_platform.c",
```

Comment:
I think this was added by mistake maybe? It's meant for the `core` integration.

Reply:
I removed this as well as a few of the other other new .c files based on fire's previous work on this.